app/react/kubernetes/volumes/ListView/column.tsx

```diff
@@ -0,0 +1,105 @@
+import { createColumnHelper } from '@tanstack/react-table';
+
+import KubernetesVolumeHelper from '@/kubernetes/helpers/volumeHelper';
+import { isoDate } from '@/portainer/filters/filters';
+
+import { Link } from '@@/Link';
+
+import { VolumeViewModel } from './types';
+import { isSystemVolume } from './isSystemVolume';
+
+const helper = createColumnHelper<VolumeViewModel>();
+
+export const columns = [
+  helper.accessor('PersistentVolumeClaim.Name', {
+    header: 'Name',
+    cell: ({ row: { original: item } }) => (
+      <>
+        <Link
+          to="kubernetes.volumes.volume"
+          params={{
+            namespace: item.ResourcePool.Namespace.Name,
+            name: item.PersistentVolumeClaim.Name,
+          }}
+        >
+          {item.PersistentVolumeClaim.Name}
+        </Link>
+        {isSystemVolume(item) ? (
+          <span className="label label-info image-tag label-margins">
```

Comment:
Please consider using `app/react/components/Badge/Badge.tsx` to keep similar to kube react tables